libs/rxidb/README.md

```diff
@@ -0,0 +1,220 @@
+<h1 align="center"> RxIDB ⚙️</h1>
+
+RxIDB is a library for storing and retrieving data from indexedDB databases, using an [RxJS](https://github.com/ReactiveX/rxjs) based API.
+
+It supports:
+
+✅ &nbsp;Creating new IndexedDB databases and Object Stores <br/>
+✅ &nbsp;Making CRUD operations on the above-mentioned stores <br/>
+✅ &nbsp;Read data out of indexedDB <br/>
+
+---
+
+<p align="center">
+  <a href="https://www.npmjs.com/package/@this-dot/rxidb"><img src="https://img.shields.io/badge/%40this--dot-%2Frxidb-blueviolet" /></a>
+  <a href="https://www.npmjs.com/package/@this-dot/rxidb"><img src="https://img.shields.io/npm/v/@this-dot/rxidb" /></a>
+  <a href="https://github.com/thisdot/open-source/actions/workflows/ci.yml?query=branch%3Amain"><img src="https://github.com/thisdot/open-source/actions/workflows/ci.yml/badge.svg" /></a>
+  <a href="https://github.com/thisdot/open-source/blob/main/LICENSE.md"><img src="https://img.shields.io/npm/l/@this-dot/rxidb" /></a>
+  <a href="https://github.com/thisdot/open-source/issues?q=is%3Aissue+is%3Aopen+label%3Arxidb"><img src="https://img.shields.io/github/issues/thisdot/open-source?q=is%3Aissue+is%3Aopen+label%3Arxidb" /></a>
+</p>
+
+---
+
+## Usage
+
+### Installation
+
+1. Install the package:  
+   `npm install @this-dot/cypress-indexeddb`  
+   or  
+   `yarn add @this-dot/cypress-indexeddb`
+
+### Connecting to an IndexedDB database and creating an Object Store
+
+The `connectIndexedDb` function is used to connect to an IndexedDB database. It takes a string as an argument, which is the name of the database. It returns an Observable that emits an `IDBDatabase` instance. Using the `getObjectStore` operator, you can create an Object Store. It also returns the store if it already exists.
+
+```typescript
+import { connectIndexedDb, getObjectStore } from '@this-dot/rxidb';
+import { Observable } from 'rxjs';
+
+// Connect to an IndexedDB database (create if the database does not exist)
+const database$: Observable<IDBDatabase> = connectIndexedDb('example_database');
+// Create an Object Store or get an existing one.
+const store$: Observable<IDBObjectStore> = database$.pipe(getObjectStore('example_store'));
+
+store$.subscribe((store) => {
+  // Do something with the store
+});
+```
+
+The `getObjectStore` function takes a string as an argument, which is the name of the Object Store. It returns an Observable that emits an `IDBObjectStore` instance. The second (optional) parameter allows you to configure your Object Store, for example, you can set it to `autoIncrement`.
```

Comment:
Are there other config options available for this, or only autoIncrement? That would be worth mentioning here, or at least pointing to where the config options available are.

Reply:
added